### PR TITLE
Feature: Template flows

### PR DESCRIPTION
**Pull request by akanshaaa19**

### CODE REVIEW

File: src/containers/Form/FormLayout.tsx [hunk 1]
```diff
@@ -578,6 +585,12 @@ export const FormLayout = ({
                 variant="outlined"
                 color="primary"
                 onClick={() => {
+                  if (additionalAction?.state === 'template') {
```
Comment (mdshamoon) [inline src/containers/Form/FormLayout.tsx:588]: If possible can we make a generic way to update this and not hardcode template inside formlayout component

File: src/containers/Flow/FlowList/FlowList.test.tsx [hunk 2]
```diff
@@ -21,15 +22,17 @@ import { Flow } from '../Flow';
...
 const mocks = [
-  getFlowCountQuery,
-  getFlowCountQuery,
-  getFlowCountQuery,
-  getFlowCountQuery,
-  filterFlowQuery,
-  filterFlowQuery,
-  filterFlowQuery,
-  filterFlowQuery,
+  getFlowCountQuery(isActiveFilter),
```
Comment (mdshamoon) [inline src/containers/Flow/FlowList/FlowList.test.tsx:28]: At this point can you check why there are multiple calls being made to filter and count query and see if you can reduce them. If it seems to take more time, do it in a separate PR

**Review by mdshamoon — CHANGES_REQUESTED** [review]

**Comment by akanshaaa19** [comment]
> 1. Ideally view it should directly open the flow. What would opening the flow in edit mode achieve.
> 
> Screenshot 2024-07-15 at 4 39 40 PM
> 2. Lets call the menu item templates since we are already inside flows.
> 
> Screenshot 2024-07-15 at 4 38 33 PM

on view it we will show description about the template flow thats why it takes to edit page first

**Review by mdshamoon — COMMENTED** [review]
Please check failing unit and integration test cases. Otherwise looks good

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**